Blocks/HTTP/impl/posix_client.h

```diff
@@ -105,9 +105,16 @@ class GenericHTTPClientPOSIX final {
         // It swallows the `SocketWriteException` exception for huge payloads, as Current's HTTP server logic
         // does intentionally close the HTTP connection prematurely if `Content-Length` exceeds a reasonable limit.
         try {
+#ifndef CURRENT_WINDOWS
           connection.BlockingWrite("Content-Length: " + std::to_string(request_body_contents_.length()) + "\r\n", true);
           connection.BlockingWrite("\r\n", true);
           connection.BlockingWrite(request_body_contents_, false);
+#else
+          // TODO(grigoriyn): this fix for the PayloadTooLarge test on Windows is temporary, need to revisit it.
+          connection.BlockingWrite("Content-Length: " + std::to_string(request_body_contents_.length()) + "\r\n\r\n" +
+                                       request_body_contents_,
+                                   false);
```

Comment:
In 3xBlockingWrite version of the code on Windows the "Content-Length" is received, analyzed on server, predefined response is sent back (in case of too large length value) and the connection is closed **before** the client finishes the last `BlockingWrite` (with the body itself). And when it tries to get the respone, the `::recv` fails with `WSAECONNABORTED`. There should be more universal solution of that, of course.